modules/axnet/src/lwip_impl/mod.rs

```diff
@@ -0,0 +1,38 @@
+mod addr;
+mod dns;
+mod driver;
+mod tcp;
+mod udp;
+
+pub use self::addr::{IpAddr, Ipv4Addr, SocketAddr};
+pub use self::dns::{dns_query, resolve_socket_addr};
+pub use self::driver::init;
+pub use self::tcp::TcpSocket;
+pub use self::udp::UdpSocket;
+use core::ffi::c_uint;
+use ruxhal::time::current_time;
+
+use axsync::Mutex;
+use lazy_init::LazyInit;
+
+static LWIP_MUTEX: LazyInit<Mutex<u32>> = LazyInit::new();
+
+const RECV_QUEUE_LEN: usize = 16;
+const ACCEPT_QUEUE_LEN: usize = 16;
+
+/// Benchmark raw socket receive bandwidth.
+pub fn bench_receive() {}
```

Comment:
Please check whether `bench_receive` and `bench_transmit` are used, as well as smoltcp_impl.

Reply:
I do not find where they are be used.